components/tableNodes/NodeTable.component.tsx

```diff
@@ -42,6 +42,26 @@ type NodeTableProps = {
   show: (visible: boolean, message: string) => void;
 };
 
+function convert(param: any): number {
```

Comment:
please can put this function inside the AppUtils?

I see no reason on why we need to avoid use AppUtils

Reply:
so sorry, I missed the review!